setup.cfg

```diff
@@ -17,6 +17,7 @@ python_requires = >=3.6
 include_package_data = True
 setup_requires = setuptools_scm
 install_requires =
+    pyqt5<5.12
```

Comment:
According to @astrofrog the `5.12` wheels have issues, so we are avoiding them. Maybe eventually this should be `!=5.12`?